tox.ini

```diff
@@ -1,4 +1,4 @@
 [flake8]
-ignore = F401,E501,E402,E401
+ignore = F401,E501,E402,E401, E305, E302, N812
```

Comment:
2) Because that's what you called it :P I will change.
3) <raspberry noise> fine :P
4) Seriously. 
5) Wait... I thought that was E302? And the reason I was removing the check for 2 blank lines is that *you* weren't following it ;) My editor bitches at me when I do it so I was. But it showed a bunch of errors when I pulled the latest saying it needed 2 lines after a class.